.changeset/tall-pots-live.md

```diff
@@ -0,0 +1,14 @@
+---
+"thirdweb": minor
+---
+
+NFT component provider changed from NFT to NFT.Provider
+
+```tsx
+<NFT.Provider contract={contract} tokenId={0n}>
+  <Suspense fallback={"Loading nft..."}>
```

Comment:
why can `NFT.Provider` not wrap suspense (internally) and you can pass the fallback to it? this seems pretty clunky...

Reply:
I agree it feels clunky, at the same time `NFT.Provider` is currently headless/UI-free which is nice. Introducing UI as props to that component feels like the wrong direction

Reply:
Noted